src/styles/main.scss

```diff
@@ -0,0 +1,10 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
```

Comment:
Consider moving general styles to a separate file to keep your main file clean and clear